debugging.rst

```diff
@@ -0,0 +1,84 @@
+.. _debugging:
+
+==============
+Debugging Jobs 
+==============
+
+Flux supports parallel debuggers such as Rogue Wave Software (RWS)'s
+TotalView parallel debugger (https://www.roguewave.com).
+
+----------------------------------
+Parallel Debugging using TotalView
+----------------------------------
+
+Debugging your MPI job from the beginning of
+parallel program execution:
+
+.. code-block:: sh
+
+  totalview --args flux mini run -N 2 -n 2 ./mpi-program
```

Comment:
Nit pick.  ReST has a `console` code-block that supports highlighting of the terminal prompt, command, and output.  Since this is just a single line, it doesn't matter all that much.  Mainly an FYI.

```suggestion
.. code-block:: console

  $ totalview --args flux mini run -N 2 -n 2 ./mpi-program
```

Reply:
Nice! Pretty powerful. Does ReST support most of GitHub mark down verbatim block ? In this case, for example ```console?